src/integration.test.ts

```diff
@@ -2,12 +2,18 @@ import { genKeyPairAndSeed, SkynetClient } from "./index";
 import { MAX_GET_ENTRY_TIMEOUT } from "./registry";
 import { MAX_REVISION } from "./utils";
 
-// TODO: Use siasky.dev when available.
-const client = new SkynetClient("https://siasky.net");
+let portal = "https://siasky.net";
+
+// To test a specific server, e.g. SKYNET_JS_INTEGRATION_TEST_SERVER=https://eu-fin-1.siasky.net yarn test src/integration.test.ts
+const testServer = process.env.SKYNET_JS_INTEGRATION_TEST_SERVER;
+if (testServer) {
+  portal = testServer;
+}
+console.log(`Running integration tests with portal ${portal}`);
```

Comment:
instead of console.log, I would wrap all of the tests in a describe block and put it there like

```javascript
describe("Integration test for ${portal}", () => {
  // all other "describe" copied here
});
```